intents/classroom/parse_locate_classroom.py

```diff
@@ -1,21 +1,33 @@
 import json
+import random
+import sys
 
-with open('intentLocateClassroomRO.json') as intentLocateClassroomRO:
-	intentRO = json.load(intentLocateClassroomRO)
-with open('intentLocateClassroomEN.json') as intentLocateClassroomEN:
-	intentEN = json.load(intentLocateClassroomEN)
+filename = ''
+output_file = ''
+if sys.argv[1] == "en":
+	print("en")
```

Comment:
Are the prints to be read by humans or something else? Cause if it's meant to be human-readable, might be better to print something like "Detected language - en". Though honestly this doesn't offer much signal so maybe just remove the prints entirely.